Content.Shared/PDA/Ringer/RingerComponent.cs

```diff
@@ -0,0 +1,57 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.PDA.Ringer;
+
+[RegisterComponent, NetworkedComponent, Access(typeof(SharedRingerSystem))]
+[AutoGenerateComponentState(true, true), AutoGenerateComponentPause]
+public sealed partial class RingerComponent : Component
```

Comment:
named args blease I beg, fieldDeltas: true. Makes future updates way easier and avoid bugs if args change.